src/services/iamUser/data.ts

```diff
@@ -171,7 +171,7 @@ const managedPoliciesByUsername = async (
   iam: IAM,
   { UserName }: User
 ): Promise<{ UserName: string; ManagedPolicies: AttachedPolicy[] }> =>
-  new Promise(resolveUserPolicies => {
+  new Promise(resolve => {
```

Comment:
Any specific reason for changing these resolve param names?

Reply:
For this specific one, is was to keep it inline with the rest of the renames on the iam resources. Looking at the rest of the code base it seems like the more common standard is to use `resolve` rather than `resolve<what>` so rather than correct the names of the resolve callbacks, I went with `resolve`
```
> grep -R 'new Promise(resolve =>' ./src | wc -l
50
```

```
> grep -E -R 'new Promise\(resolve.+ =>' ./src | wc -l
11
```

If one way or the other preferred, I can update accordingly.

Reply:
I think thats fine, thanks for the clarification!